content/minfraud/api-documentation/_schemas/ResponseIpAddress.mdx

```diff
@@ -11,11 +11,16 @@ import responseJson from '../_examples/response';
   with four modifications:
 
 1. `risk` has been added directly to the `ip_address` object
-1. `local_time` has been added to the `location` sub-object
-1. The `maxmind` object is not present. See below for descriptions.
-1. minFraud Insights and Factors return the following anonymous IP outputs: _
-   `is_anonymous` _ `is_anonymous_vpn` _ `is_hosting_provider` _
-   `is_public_proxy` _ `is_residential_proxy` _ `is_tor_exit_node`
+2. `local_time` has been added to the `location` sub-object
+3. The `maxmind` object is not present. See below for descriptions.
+4. minFraud Insights and Factors return the following anonymous IP outputs:
+
+- `is_anonymous`
+- `is_anonymous_vpn`
+- `is_hosting_provider`
+- `is_public_proxy`
+- `is_residential_proxy`
+- `is_tor_exit_node`
```

Comment:
@oschwald @mm-jpoole 

<img width="819" alt="Screen Shot 2022-09-30 at 6 48 44 PM" src="https://user-images.githubusercontent.com/52248962/193366404-161b1e09-b9dc-46ba-b09c-c434c9ee887a.png">

I tried applying the following, but it reverts to the above and as a result doesn't look like what it should here: https://dev.maxmind.com/minfraud/api-documentation/responses?lang=en#schema--response--ip-address:

```
diff --git a/content/minfraud/api-documentation/_schemas/ResponseIpAddress.mdx b/content/minfraud/api-documentation/_schemas/ResponseIpAddress.mdx
index f42fd4f..3bce3e4 100644
--- a/content/minfraud/api-documentation/_schemas/ResponseIpAddress.mdx
+++ b/content/minfraud/api-documentation/_schemas/ResponseIpAddress.mdx
@@ -15,12 +15,12 @@ import responseJson from '../_examples/response';
 3. The `maxmind` object is not present. See below for descriptions.
 4. minFraud Insights and Factors return the following anonymous IP outputs:

-- `is_anonymous`
-- `is_anonymous_vpn`
-- `is_hosting_provider`
-- `is_public_proxy`
-- `is_residential_proxy`
-- `is_tor_exit_node`
+  - `is_anonymous`
+  - `is_anonymous_vpn`
+  - `is_hosting_provider`
+  - `is_public_proxy`
+  - `is_residential_proxy`
+  - `is_tor_exit_node`
```

Reply:
Sorry, never mind. 2 indents did the trick in https://github.com/maxmind/dev-site/pull/826/commits/a4455594a887274123027cd3151fe30c0aec4e97.